discourse-mixin/panels.libsonnet

```diff
@@ -50,16 +50,16 @@ local commonlib = import 'common-lib/common/main.libsonnet';
       + g.panel.timeSeries.panelOptions.withDescription('The median amount of time for "topics show" requests for the selected site.')
       + g.panel.timeSeries.standardOptions.withUnit('s'),
 
-    latest99thPercentileRequestTime:
-      g.panel.histogram.new('Latest request time')
-      + g.panel.histogram.queryOptions.withTargets([signals.overview.latest99thPercentileRequestTime.asTarget()])
-      + g.panel.histogram.panelOptions.withDescription('The 99th percentile amount of time for "latest" page requests for the selected site.')
+    latestRequestTimePercentiles:
+      g.panel.histogram.new('Latest request time percentiles')
```

Comment:
As we just discussed on the CouchDB mixin update, for these where we split by instance we should stack it so it shows properly